### DISCUSSION TITLE
Group modules

### DISCUSSION THREAD

**Discussion by rubiin** [Jas-SinghFSU/HyprPanel#332] [Q&A] [answered]
Is there a way to group modules similar to the circled region on screenshot

This is on waybar, wanted to achiever somthing similar
![image](https://github.com/user-attachments/assets/13d3d18c-3f3f-46bb-9a35-1e6134b2dcd6)

**Comment by rubiin** [comment]
want to group these in a single overlay similar to above
![image](https://github.com/user-attachments/assets/c859ad0a-03c1-4c11-982d-dda3c098822c)

**Comment by Jas-SinghFSU** [comment, marked as answer]
Not currently but is a feature I've wanted to implement. Make a request in the issues when you get a chance please.

**Reply by rubiin** [reply]
sure thing. could you also look into the pr as well.
I have just getting started on the repo code and will likely send in more
